packages/back-end/src/api/openapi/schemas/FeatureRevision.yaml

```diff
@@ -0,0 +1,33 @@
+type: object
+required:
+  - baseVersion
+  - version
+  - comment
+  - date
+  - status
+  - rules
+properties:
```

Comment:
I noticed in the screenshot, `organization` and possibly other fields are also being returned. We should restrict the fields to just what is listed here. Might need to specify `additionalProperties: false` here in the JSON schema to make Typescript detect this and throw type errors.

Reply:
`additionalProperties: false` doesn't seem to do anything here. I'll take a deeper dive though...